engine/src/main/java/org/terasology/world/generation/World.java

```diff
@@ -28,6 +28,11 @@
 
     Region getWorldData(Region3i region);
 
+    /**
+     * @return the sea level, measured in blocks
```

Comment:
Maybe add an extra note here to point out what's magic about the sea level? Currently that it is use as the sole reflection layer

Like: "the sea level, measured in blocks. May be used for setting such things as the block layer to use in water reflections"